### ISSUE TITLE
Email on public entry

### ISSUE THREAD

**Issue by hl** [uberous/lista#13]
When someone submits a public entry the admin should receive an email.

Currently there is only 1 user but you can easily create more. Should we all send them an email or should we create roles? I think keeping this app as simple as possible would be the goal.

A very easy mailer would be https://github.com/thoughtbot/bamboo

**Comment by jeroenvisser101** [comment]
Or maybe just a slack notification?

**Comment by hl** [comment]
I like the slack notification but I think an email is more generic, otherwise we are 'forcing' people to use slack while everyone has an email address.

**Comment by jeroenvisser101** [comment]
> otherwise we are 'forcing' people to use slack while everyone has an email address.

agreed but these people would be admins, I think we could do slack for now since it's super easy too, if it ever becomes an issue, we can do it differently.

**Comment by hl** [comment]
I still think email will be easier for instahustlers.com because Dee will most likely run it and she has no idea what slack is
